packages/xsnap/test/test-boot-lockdown.js

```diff
@@ -7,6 +7,8 @@ import * as proc from 'child_process';
 import * as os from 'os';
 import * as fs from 'fs';
 
+import { bundlePaths } from '@agoric/xsnap-lockdown';
```

Comment:
Does `getLockdownBundle()` somehow not suffice here? hm.

Reply:
removed